lib/engine/game/g_18_al.rb

```diff
@@ -12,6 +12,13 @@ class G18AL < Base
       GAME_LOCATION = 'Alabama, USA'
       GAME_RULES_URL = 'http://www.diogenes.sacramento.ca.us/18AL_Rules_v1_64.pdf'
       GAME_DESIGNER = 'Mark Derrick'
+
+      def setup
+        @companies.each do |company|
```

Comment:
you can create a module that does this and reduce all of these codes to

def setup
  mark_derrick or some better generic name